src/globals.css

```diff
@@ -27,6 +27,10 @@
     .bg-overlay {
         background: rgba(20, 23, 24, 0.5);
     }
+
+    .tab-shadow {
+        box-shadow: 0px 16px 16px -4px rgba(0, 0, 0, 0.1);
```

Comment:
I think it would be better to either add this to the tailwind definitions if its going to be used elsewhere, or if not then define it inline in tailwind. We really don't want to pollute the global namespace if we can avoid it